server/xpub/entities/manuscript/index.js

```diff
@@ -0,0 +1,114 @@
+const lodash = require('lodash')
+const emptyManuscript = require('./helpers/empty')
+const dataAccess = require('./data-access')
+
+const mergeObjects = (...inputs) =>
+  lodash.mergeWith(
+    ...inputs,
+    // always replace arrays instead of merging
+    (objValue, srcValue) => {
+      if (lodash.isArray(srcValue)) {
+        return srcValue
+      }
+      return undefined
+    },
+  )
+
+const Manuscript = {
+  MAX_SUGGESTED_REVIEWERS: 6,
+  MIN_SUGGESTED_REVIEWERS: 3,
+
+  find: dataAccess.selectById,
+  all: dataAccess.selectAll,
+  findByStatus: dataAccess.selectByStatus,
+
+  new: () => lodash.cloneDeep(emptyManuscript),
```

Comment:
`Manuscript.new` seems like it would be better placed to be `Manuscript.create`.

Maybe there should be a `Manuscript` class, where some of these functions return instances of said class. It means we can avoid cloning objects.

```
class Manuscript {
  constructor() {
    // instantiate empty manuscript
  },
  addTeam(roles) {},
  checkPermission() {},
  removeOptionalBlankReviewers() {}
}

const Entity = {
   ...
   create: () => new Manuscript()
   ...
}
```

Is this what you meant by the duplicate code that could be refactored out later?